### ISSUE TITLE
STAR index generation or mapping with --sjfbGTFfile

### ISSUE THREAD

**Issue by pansapiens** [MonashBioinformaticsPlatform/RNAsik-pipe#51]
I noticed RNAsik does not use the`--sjfbGTFfile` option (and associated `--sjdb*` options). 

Given that the GTF is usually available, does it make sense to include this either at the index generation or mapping step ? I don't have a good sense as to how much it improves mapping around splice junctions, but the manual suggests it could make a big difference.

**Comment by drpowell** [comment]
Yes, it probably should.  Either using that, or doing a 2-pass alignment will give better slice junction alignments

**Comment by serine** [comment]
Maybe it should be included. The easiest way would be to start by only allowing splice junction detection with Ensembl reference. will need to set these flags

- `--sjdbGTFchrPrefix` 
- `--sjdbOverhang`
